build.gradle

```diff
@@ -672,6 +672,7 @@ project(':streams') {
   archivesBaseName = "kafka-streams"
 
   dependencies {
+    compile project(':core')
```

Comment:
@enothereska that's probably the easiest solution. Worth checking what @guozhangwang thinks.